pkg/plugins/config/config.go

```diff
@@ -21,7 +21,7 @@ type Cfg struct {
 	// AWS Plugin Auth
 	AWSAllowedAuthProviders []string
 	AWSAssumeRoleEnabled    bool
-
+	awsExternalId           string
```

Comment:
this is not used atm. If this externalId is something that needs to be used by external plugins you need to expose it and set it as the other variables:

https://github.com/grafana/grafana/blob/main/pkg/plugins/envvars/envvars.go#L87

Reply:
hmm great catch! Weirdly I can get it to work in other external plugins, but I guess that's because of this line in sessions.go maybe: ```err = os.Setenv(awsds.GrafanaAssumeRoleExternalIdKeyName, awsPluginSec.Key("external_id").Value())```

I think I can remove that line though if we use the function you have shared. I'll take that approach instead.